mycroft/skills/intent_service.py

```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 #
 import time
-
+from builtins import range
```

Comment:
Can we move this by `from past.builtins` and maybe make it explicitly `from future.builtins`?

Reply:
Sure, `future.builtins.range is builtins.range` evaluates to True so shouldn't be an issue